### DISCUSSION TITLE
I would like to convert all selected rows into unselected rows with a button

### DISCUSSION THREAD

**Discussion by thsnyzkn** [TanStack/table#2463] [General]
Hey everyone,

I have created a table by following through this example(https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/row-selection).

I just want to add a button which is going to be outside of the table component and when I click this button, all the selected rows will be unselected. 

I even couldn't wrap my head around to come up with an idea or which function of react-table that I need to use.

Thanks!

**Comment by mkosucu-wr** [comment]
I think `toggleAllRowsSelected` does what you need on https://github.com/tannerlinsley/react-table/blob/master/docs/api/useRowSelect.md. I have not tried it but I will try to implement something this afternoon.
